webapp/src/shared/components/Input/EditUserForm.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+import { Button, ButtonVariant, Input, InputVariant } from '../';
+import './EditUserForm.css';
+
+export default function EditUserForm() {
+  const [fullName, setFullName] = useState('');
+  const [email, setEmail] = useState('');
+
+  const buttonAction = (e: React.FormEvent<HTMLFormElement>): void => {
+    e.preventDefault();
+    alert(`Your name: "${fullName}" and your email: "${email}" were saved.`);
+  };
```

Comment:
suggestion: I know this is not the final version, but we probably want to reset the inputs after processing the form data.
```
setFullName('');
setEmail('');
```

Reply:
Done